test/e2e/specs/site-editor/site-editor-export.spec.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Site Editor Templates Export', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await Promise.all( [
+			requestUtils.activateTheme( 'emptytheme' ),
+			requestUtils.deleteAllTemplates( 'wp_template' ),
+			requestUtils.deleteAllTemplates( 'wp_template_part' ),
+		] );
+	} );
+
+	test.afterAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'twentytwentyone' );
+	} );
+
+	test( 'clicking export should download emptytheme.zip file', async ( {
+		admin,
+		editor,
+		page,
+	} ) => {
+		await admin.visitSiteEditor( {
+			postId: 'emptytheme//index',
+			postType: 'wp_template',
+		} );
+		await editor.canvas.locator( 'body' ).click();
```

Comment:
I know what this does, but I guess it might be obscure to others. 😄 Should we make a util for it, e.g. `siteEditor.enterEditMode()`? Or just a simple comment would do for now. What do you think?

Reply:
We can also use `canvas=edit` query argument - [459ace7](https://github.com/WordPress/gutenberg/pull/56675/commits/459ace7da6e06705121172162cc7a72af9d59637).